src/Microsoft.Data.SqlClient/tests/ManualTests/SQL/ParameterTest/DateTimeVariantTest.cs

```diff
@@ -102,7 +102,7 @@ private static void TestSimpleParameter_Type(object paramValue, string expectedT
             catch (Exception e)
             {
                 if (IsExpectedException(e, paramValue, expectedTypeName, expectedBaseTypeName))
-                    LogMessage(tag, "[EXPECTED EXPECTION] " + e.Message);
+                    LogMessage(tag, "[EXPECTED EXCEPTION] " + e.Message);
```

Comment:
Please also update this is [BSL files](https://github.com/dotnet/SqlClient/search?q=EXPECTION&unscoped_q=EXPECTION)

Reply:
should be done now